core/state/database.go

```diff
@@ -183,14 +192,6 @@ type TrieDbState struct {
 }
 
 func NewTrieDbState(root common.Hash, db ethdb.Database, blockNr uint64) (*TrieDbState, error) {
```

Comment:
Now there is no reason to have the second return value (error). Can clean it up in a separate PR